src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -11,16 +10,17 @@
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import java.util.concurrent.TimeUnit;
+import com.wikia.webdriver.common.contentpatterns.XSSContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
 
 /**
  * Set of commonly used actions invoked by executing JavaScript on a web page
  */
 public class JavascriptActions {
 
-  private JavascriptExecutor js;
-  private WebDriver driver;
-  private static int WEBDRIVER_WAIT_TIMEOUT_SEC = 15;
+  private final static int WEBDRIVER_WAIT_TIMEOUT_SEC = 15;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)